ggshield/verticals/secret/secret_scan_collection.py

```diff
@@ -19,6 +22,41 @@ class Result(NamedTuple):
     file: Scannable  # filename that was scanned
     scan: ScanResult  # Result of content scan
 
+    def enrich_matches(self):
+        content = self.content
+        is_patch = self.filemode != Filemode.FILE
+        lines = get_lines_from_content(content, self.filemode)
+        for policy_break in self.scan.policy_breaks:
+            res = []
+            for match in policy_break.matches:
+                if match.index_start is None or match.index_end is None:
+                    res.append(match)
+                    continue
+                index_minified_start = match.index_start
+                index_minified_end = match.index_end
```

Comment:
What are those two? They don't exist in ExtendedMatch.

Reply:
Ouf this is just a remain of another attempt a working refacto...